src/player_components.rs

```diff
@@ -171,9 +251,11 @@ mod tests {
     fn bet_all_in() {
         let mut player = Player::new(0);
         player.deal([Card::new(Rank::Ace, Suit::Clubs), Card::new(Rank::Ace, Suit::Hearts)]);
-        player.bet(DEFAULT_START_MONEY);
-        player.bet(DEFAULT_START_MONEY);
-        player.bet(DEFAULT_START_MONEY);
+        assert_eq!(player.bet(DEFAULT_START_MONEY + 3), DEFAULT_START_MONEY);
```

Comment:
Might want to keep in mind that if the max bet logic is added to the Player.bet(), it could impact these tests.

Reply:
Yeah currently I think the table controls max bets not the players, since the table owns the pot. Does that seem like reasonable design?

Reply:
It would break other tests. I’ll add it as an issue.